### DISCUSSION TITLE
External libraries - auto scan?

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#8767] [Q&A]
Is it intended that after restart of immich, the scan of external libraries needs to be triggered manually again if not finished?
Or is there some setting for automatic rescans of the external libraries?

**Comment by aviv926** [comment]
https://immich.app/docs/features/libraries#set-custom-scan-interval

**Comment by rlue** [comment]
Does anyone have, say, a scripted solution that utilizes inotify to trigger a scan over the API when files are added or removed from an external library? My holy grail is to have Immich automatically update the library the moment new files are added, without setting the scan interval to something ridiculously low like 10 seconds.
